packages/chat-client/test/client/client.spec.ts

```diff
@@ -54,38 +55,71 @@ describe("ChatClient", () => {
   });
 
   it("can register an account on the keyserver", async () => {
-    const publicKey = await client.register({
-      account: TEST_CLIENT_ACCOUNT,
+    const walletSelf = Wallet.createRandom();
+    const walletPeer = Wallet.createRandom();
+    const publicKeySelf = await client.register({
+      account: `eip155:1:${walletSelf.address}`,
+      onSign: (message) => walletSelf.signMessage(message),
     });
-    const peerPublicKey = await peer.register({
-      account: TEST_PEER_ACCOUNT,
+
+    const publicKeyPeer = await client.register({
+      account: `eip155:1:${walletPeer.address}`,
+      onSign: (message) => walletPeer.signMessage(message),
     });
 
-    expect(publicKey.length).toBeGreaterThan(0);
-    expect(peerPublicKey.length).toBeGreaterThan(0);
+    expect(publicKeySelf.length).toBeGreaterThan(0);
+    expect(publicKeyPeer.length).toBeGreaterThan(0);
   });
 
   it("can resolve an account on the keyserver", async () => {
-    const publicKey = await peer.resolve({
-      account: TEST_CLIENT_ACCOUNT,
+    const walletSelf = Wallet.createRandom();
+    const walletPeer = Wallet.createRandom();
+
+    const peerIdentityPublicKey = await peer.register({
+      account: `eip155:1:${walletPeer.address}`,
+      onSign: (message) => walletPeer.signMessage(message),
+    });
+
+    const selfIdentityPublicKey = await client.register({
+      account: `eip155:1:${walletSelf.address}`,
+      onSign: (message) => walletSelf.signMessage(message),
+    });
+
+    const selfIdentityCacao = await peer.resolveIdentity({
+      publicKey: `${selfIdentityPublicKey}`,
     });
-    const peerPublicKey = await client.resolve({
-      account: TEST_PEER_ACCOUNT,
+    const peerIdentityCacao = await client.resolveIdentity({
+      publicKey: `${peerIdentityPublicKey}`,
     });
 
-    expect(publicKey.length).toBeGreaterThan(0);
-    expect(peerPublicKey.length).toBeGreaterThan(0);
+    expect(selfIdentityCacao.p.iss).toEqual(
+      `did:pkh:eip155:1:${walletSelf.address}`
+    );
+    expect(peerIdentityCacao.p.iss).toEqual(
+      `did:pkh:eip155:1:${walletPeer.address}`
+    );
   });
 
   it("can send & receive invites", async () => {
+    const walletSelf = Wallet.createRandom();
+    const walletPeer = Wallet.createRandom();
     let peerReceivedInvite = false;
     let peerJoinedChat = false;
 
-    const peerInvitePublicKey = await peer.register({
-      account: TEST_PEER_ACCOUNT,
+    console.log("Attempting to regjuster CLIENT");
```

Comment:
Marking the logs here for removal based on assumption we don't want these once this PR is out of draft stage.

```suggestion

```